hadoop-yarn-project/hadoop-yarn/hadoop-yarn-common/src/main/resources/webapps/static/customjs/gridjs.umd.js

```diff
@@ -0,0 +1,24 @@
,e?x("div",{ref:i,className:rt(et("head"),u.className.header),style:r({},u.style.header)},x(Gt,{position:t.PluginPosition.Header})):null}function In(){var n=bt(null),e=gt(!0),o=e[0],i=e[1],u=It();return yt(function(){0===n.current.children.length&&i(!1)},[n]),o?x("div",{ref:n,className:rt(et("footer"),u.className.footer),style:r({},u.style.footer)},x(Gt,{position:t.PluginPosition.Footer})):null}function Tn(){var t=It(),n=jt().dispatch,e=Ht(function(t){return t.status}),o=Ht(function(t){return t.data}),i=Ht(function(t){return t.tableRef}),u={current:null},s=xn(function(){try{n(function(t){return r({},t,{status:l.Loading})});var e=function(e,o){try{var i=Promise.resolve(t.pipeline.process()).then(function(t){n(function(t){return function(n){return t?r({},n,{data:t,status:l.Loaded}):n}}(t)),setTimeout(function(){n(function(t){return t.status===l.Loaded?r({},t,{status:l.Rendered}):t})},0)})}catch(t){return o(t)}return i&&i.then?i.then(void 0,o):i}(0,function(t){Vt.error(t),n(function(t){return r({},t,{data:null,status:l.Error})})});return Promise.resolve(e&&e.then?e.then(function(){}):void 0)}catch(t){return Promise.reject(t)}},t.processingThrottleMs);return yt(function(){return n(Nn(t.header)),s(),t.pipeline.on("updated",s),function(){return t.pipeline.off("updated",s)}},[]),yt(function(){t.header&&e===l.Loaded&&null!=o&&o.length&&n(Nn(t.header.adjustWidth(t,i,u)))},[o,t,u]),x("div",{role:"complementary",className:rt("gridjs",et("container"),e===l.Loading?et("loading"):null,t.className.container),style:r({},t.style.container,{width:t.width})},e===l.Loading&&x("div",{className:et("loading-bar")}),x(En,null),x("div",{className:et("wrapper"),style:{height:t.height}},x(Cn,null)),x(In,null),x("div",{ref:u,id:"gridjs-temp",className:et("temp")}))}var Ln=/*#__PURE__*/function(t){function n(n){var e;return(e=t.call(this)||this).config=void 0,e.plugin=void 0,e.config=(new pn).assign({instance:u(e),eventEmitter:u(e)}).update(n),e.plugin=e.config.plugin,e}o(n,t);var e=n.prototype;return e.updateConfig=function(t){return this.config.update(t),this},e.createElement=function(){return x(fn.Provider,{value:this.config,children:x(Tn,{})})},e.forceRender=function(){return this.config&&this.config.container||Vt.error("Container is empty. Make sure you call render() before forceRender()",!0),this.destroy(),q(this.createElement(),this.config.container),this},e.destroy=function(){this.config.pipeline.clearCache(),q(null,this.config.container)},e.render=function(t){return t||Vt.error("Container element cannot be null",!0),t.childNodes.length>0?(Vt.error("The container element "+t+" is not empty. Make sure the container is empty and call render() again"),this):(this.config.container=t,q(this.createElement(),t),this)},n}(Q);t.Cell=X,t.Component=S,t.Config=pn,t.Grid=Ln,t.Row=Z,t.className=et,t.createElement=x,t.createRef=function(){return{current:null}},t.h=x,t.html=G,t.useConfig=It,t.useEffect=yt,t.useRef=bt,t.useSelector=Ht,t.useState=gt,t.useStore=jt,t.useTranslator=At});
```

Comment:
Thanks @riya2305 for the change.
I think we should update are licences with this new dependency, and remove the deleted ones